CRM/Admin/Form/Preferences/Mailing.php

```diff
@@ -36,6 +36,86 @@
  */
 class CRM_Admin_Form_Preferences_Mailing extends CRM_Admin_Form_Preferences {
 
+  public function preProcess() {
```

Comment:
@twomice I think the changes to this form are not related to your PR but are merge mess?

We have removed this chunk of code from core (ideally we would also remove the rest of the form & jut use the Generic form but we'd need to move those post actions to metadata per the todo)

Reply:
Dang it, I though I had it this time. Trying again...

Reply:
Ok, that's removed @eileenmcnaughton . I'll ping again when jenkins tests are done. Thanks!